usageOutput

```diff
@@ -0,0 +1 @@
+[{"count":16444}]
```

Comment:
Looks like this extra file was copied erroneously so you probably want to remove this.